### PR TITLE
network : new struct of module udp_client

### PR DESCRIPTION
**Pull request by RocioRojas**
### Contribution description
New struct of module udp_client, I added some modules like :
```

USEMODULE += gnrc_ipv6_default
USEMODULE += ipv6_addr
```

### Testing procedure
look at  firmware/network/udp_client
In firmware/Makefile need 
```
USEMODULE += udp_client
EXTERNAL_MODULE_DIRS += network
```

### Issues/PRs references

none

### CODE REVIEW

File: boards/m4a-24g/include/board.h [hunk 1]
```diff
@@ -25,7 +25,7 @@
 #include "periph_conf.h"
 #include "periph_cpu.h"
 #include "periph/cpuid.h"
-#include "uniqueid.h"
+//#include "uniqueid.h"
```
Comment (luisan00) [inline boards/m4a-24g/include/board.h:28]: Dead code should be removed

File: boards/m4a-mb/include/board.h [hunk 2]
```diff
@@ -22,7 +22,7 @@
 #include "periph_conf.h"
 #include "periph_cpu.h"
 #include "periph/cpuid.h"
-#include "uniqueid.h"
+//#include "uniqueid.h"
```
Comment (luisan00) [inline boards/m4a-mb/include/board.h:25]: Same here

**Review by luisan00 — CHANGES_REQUESTED** [review]
Remove all unused or dead code

File: firmware/network/Makefile [hunk 3]
```diff
@@ -0,0 +1,5 @@
+ifneq (,$(filter udp_client,$(USEMODULE)))
+  DIRS += udp_client
+endif
+
+include $(RIOTBASE)/Makefile.base
```
Comment (xkevin190) [inline firmware/network/Makefile:5]: EOF Line

File: firmware/network/udp_client/Makefile.dep [hunk 4]
```diff
@@ -0,0 +1,4 @@
+USEMODULE += gnrc_sock_udp
+USEMODULE += gnrc_udp
+USEMODULE += gnrc_ipv6_default
+USEMODULE += ipv6_addr
```
Comment (xkevin190) [inline firmware/network/udp_client/Makefile.dep:4]: EOF Line

**Review by xkevin190 — CHANGES_REQUESTED** [review]

File: firmware/network/udp_client/Makefile.dep (hunk 4 above)
Comment (luisan00) [inline firmware/network/udp_client/Makefile.dep:4]: All this modules are needed?

**Review by luisan00 — CHANGES_REQUESTED** [review]
First round

**Review by luisan00 — APPROVED** [review]
Looks good for me 👍

**Pull request merged**